packages/compiler/src/core/program.ts

```diff
@@ -910,3 +922,41 @@ function resolveOptions(options: CompilerOptions): CompilerOptions {
     outputPath: outputDir,
   };
 }
+
+async function emit(emitter: EmitterRef, options: CompilerOptions = {}, program: Program) {
+  const emitterName = emitter.metadata.name ?? "";
+  const outputRelativePath = options.listFiles
+    ? `./${getRelativePathFromDirectory(program.projectRoot, emitter.emitterOutputDir, false)}/`
+    : "";
+  addChildLog(`✓ ${emitterName}\t${outputRelativePath}`);
```

Comment:
Is there any way to exclude icon in the message (like the check icon above)?

In typespec vescode extension, the vscode terminal may not be able to display this.

Reply:
Those are just regular characters that work in every other terminals can you double check if they are not working, that would be quite surprising

Reply:
I checked this and it was working as expected, resolving this conversation .